cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1642,7 +1642,8 @@
                 "autoscaling:SetDesiredCapacity",
                 "autoscaling:UpdateAutoScalingGroup",
                 "autoscaling:DescribeTags",
-                "autoscaling:SetInstanceHealth"
+                "autoscaling:SetInstanceHealth",
+                "autoscaling:EnableMetricsCollection"
```

Comment:
This is probably needed in the UserRole not in the RootRole

Reply:
Yep https://github.com/awsdocs/aws-parallelcluster-user-guide/pull/9